pycloudlib/azure/cloud.py

```diff
@@ -452,21 +459,27 @@ def delete_image(self, image_id):
                 "Error deleting %s. Request returned %d", image_id, resp_code
             )
 
+    def _get_image(self, release, image_map):
+        release = image_map.get(release)
+        if release is None:
```

Comment:
In failure paths, `release` is getting reset to None in the case that we don't find it. So the error message ends up being unhelpful.

*** ValueError: No Ubuntu image found for None. Expected one of: xenial bionic focal impish jammy.


```suggestion
        image_id = image_map.get(release)
        if image_id  is None:
```